pkgs/tools/networking/debian-hostname/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "debian-hostname";
+  version = "3.23";
+
+  src = fetchurl {
+    url = "http://deb.debian.org/debian/pool/main/h/hostname/hostname_${version}.tar.gz";
+    sha256 = "sha256-vG0ZVLIoSYaf+LKmAuOfCLFwL2htS1jdeSfN61tIdu8=";
+  };
+
+  outputs = ["out"];
+
+  patches = [];
+
+  nativeBuildInputs = [];
+  buildInputs = [];
+  doCheck = false;
```

```suggestion
  doCheck = false;
```

Please add a comment why tests are disabled.